backend/game_logic/game_logic_controller.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
+from os import listdir
+import numpy as np
+
+from flask import Blueprint, request, jsonify, send_from_directory
+from backend.game_logic.game_state_elements import GameStateManager
+import os
+
+game_logic_bp = Blueprint('game-logic', __name__)
+
+backend_dir = Path(__file__).resolve().parent.parent
+IMAGES_FOLDER = backend_dir.joinpath('images').joinpath('processed')
+
+
+@game_logic_bp.route('/images-list', methods=['GET'])
```

Comment:
Would make sense for the endpoint to just be `GET /images` for the list of all images and then `GET /images/{fileName}` to get the specific image. And maybe `POST /images` to upload the image?